### ISSUE TITLE
[Bug]: Using the default version causes image pull failures

### ISSUE THREAD

**Issue by iress-ac** [cert-manager/aws-privateca-issuer#231]
### Describe the expected outcome

When using the chart without specifying a version number, the latest version should be deployed.

### Describe the actual outcome

The chart attempts to pull a malformed image version and the deployment fails.

The image it's trying to pull is: `cert-manager-aws-privateca-issuer:vv1.2.3`

[There's code here to specify the default version should be prefixed with a v](https://github.com/cert-manager/aws-privateca-issuer/blob/main/charts/aws-pca-issuer/templates/deployment.yaml#L38)
The helm chart downloads with a v in the version number by default:

**Chart.yaml:**
```
appVersion: v1.2.3
```

Hence 2 v's.

The workaround is to specify the chart version explicitly

### Steps to reproduce

1. Attempt to use the chart but do not specifying a version to deploy the latest version of the `aws-privateca-issuer`.
2. Deployment should fail pulling the image due to the malformed version

### Version

1.2.3

### Have you tried the following?

- [X] Check the [Troubleshooting section](../#troubleshooting)
- [X] Search open [issues](https://github.com/cert-manager/aws-privateca-issuer/issues)

### Category

Supported Workflow Broken

### Severity

Severity 2

**Comment by divyansh-gupta** [comment]
Thanks for opening the bug. I am working on trying to reproduce.

**Comment by divyansh-gupta** [comment]
```
guptadiv@147ddacf5011 ~: helm install awspca/aws-privateca-issuer --generate-name
NAME: aws-privateca-issuer-1670425632
LAST DEPLOYED: Wed Dec  7 10:07:14 2022
NAMESPACE: default
STATUS: deployed
REVISION: 1
TEST SUITE: None

guptadiv@147ddacf5011 ~: helm list
NAME                           	NAMESPACE	REVISION	UPDATED                             	STATUS  	CHART                     	APP VERSION
aws-privateca-issuer-1670425632	default  	1       	2022-12-07 10:07:14.687668 -0500 EST	deployed	aws-privateca-issuer-1.2.3	v1.2.3 

guptadiv@147ddacf5011 ~: kubectl get pods
NAME                                               READY   STATUS         RESTARTS   AGE
aws-privateca-issuer-1670425632-7fbff57b48-gbklx   0/1     ErrImagePull   0          92s

guptadiv@147ddacf5011 ~: kubectl describe pods
Name:         aws-privateca-issuer-1670425632-7fbff57b48-gbklx
Namespace:    default
Priority:     0
Node:         kind-control-plane/172.18.0.2
Start Time:   Wed, 07 Dec 2022 10:07:14 -0500
Labels:       app.kubernetes.io/instance=aws-privateca-issuer-1670425632
              app.kubernetes.io/name=aws-privateca-issuer
              pod-template-hash=7fbff57b48
Annotations:  <none>
Status:       Pending
IP:           10.244.0.9
IPs:
  IP:           10.244.0.9
Controlled By:  ReplicaSet/aws-privateca-issuer-1670425632-7fbff57b48
Containers:
  aws-privateca-issuer:
    Container ID:  
    Image:         public.ecr.aws/k1n1h4h4/cert-manager-aws-privateca-issuer:vv1.2.3
    Image ID:      
    Port:          8080/TCP
    Host Port:     0/TCP
    Command:
      /manager
    Args:
      --leader-elect
    State:          Waiting
      Reason:       ImagePullBackOff
    Ready:          False
    Restart Count:  0
    Liveness:       http-get http://:8081/healthz delay=15s timeout=1s period=20s #success=1 #failure=3
    Readiness:      http-get http://:8081/healthz delay=5s timeout=1s period=10s #success=1 #failure=3
    Environment:    <none>
    Mounts:
      /var/run/secrets/kubernetes.io/serviceaccount from kube-api-access-7hw6s (ro)
Conditions:
  Type              Status
  Initialized       True 
  Ready             False 
  ContainersReady   False 
  PodScheduled      True 
Volumes:
  kube-api-access-7hw6s:
    Type:                    Projected (a volume that contains injected data from multiple sources)
    TokenExpirationSeconds:  3607
    ConfigMapName:           kube-root-ca.crt
    ConfigMapOptional:       <nil>
    DownwardAPI:             true
QoS Class:                   BestEffort
Node-Selectors:              <none>
Tolerations:                 node.kubernetes.io/not-ready:NoExecute op=Exists for 300s
                             node.kubernetes.io/unreachable:NoExecute op=Exists for 300s
Events:
  Type     Reason     Age                  From               Message
  ----     ------     ----                 ----               -------
  Normal   Scheduled  2m33s                default-scheduler  Successfully assigned default/aws-privateca-issuer-1670425632-7fbff57b48-gbklx to kind-control-plane
  Warning  Failed     2m22s                kubelet            Failed to pull image "public.ecr.aws/k1n1h4h4/cert-manager-aws-privateca-issuer:vv1.2.3": rpc error: code = Unknown desc = failed to pull and unpack image "public.ecr.aws/k1n1h4h4/cert-manager-aws-privateca-issuer:vv1.2.3": failed to resolve reference "public.ecr.aws/k1n1h4h4/cert-manager-aws-privateca-issuer:vv1.2.3": failed to do request: Head "https://public.ecr.aws/v2/k1n1h4h4/cert-manager-aws-privateca-issuer/manifests/vv1.2.3": dial tcp: lookup public.ecr.aws on 172.18.0.1:53: read udp 172.18.0.2:34250->172.18.0.1:53: i/o timeout
  Warning  Failed     118s                 kubelet            Failed to pull image "public.ecr.aws/k1n1h4h4/cert-manager-aws-privateca-issuer:vv1.2.3": rpc error: code = Unknown desc = failed to pull and unpack image "public.ecr.aws/k1n1h4h4/cert-manager-aws-privateca-issuer:vv1.2.3": failed to resolve reference "public.ecr.aws/k1n1h4h4/cert-manager-aws-privateca-issuer:vv1.2.3": failed to do request: Head "https://public.ecr.aws/v2/k1n1h4h4/cert-manager-aws-privateca-issuer/manifests/vv1.2.3": dial tcp: lookup public.ecr.aws on 172.18.0.1:53: read udp 172.18.0.2:51414->172.18.0.1:53: i/o timeout
  Warning  Failed     85s                  kubelet            Failed to pull image "public.ecr.aws/k1n1h4h4/cert-manager-aws-privateca-issuer:vv1.2.3": rpc error: code = Unknown desc = failed to pull and unpack image "public.ecr.aws/k1n1h4h4/cert-manager-aws-privateca-issuer:vv1.2.3": failed to resolve reference "public.ecr.aws/k1n1h4h4/cert-manager-aws-privateca-issuer:vv1.2.3": failed to do request: Head "https://public.ecr.aws/v2/k1n1h4h4/cert-manager-aws-privateca-issuer/manifests/vv1.2.3": dial tcp: lookup public.ecr.aws on 172.18.0.1:53: read udp 172.18.0.2:52962->172.18.0.1:53: i/o timeout
  Normal   Pulling    44s (x4 over 2m32s)  kubelet            Pulling image "public.ecr.aws/k1n1h4h4/cert-manager-aws-privateca-issuer:vv1.2.3"
  Warning  Failed     34s (x4 over 2m22s)  kubelet            Error: ErrImagePull
  Warning  Failed     34s                  kubelet            Failed to pull image "public.ecr.aws/k1n1h4h4/cert-manager-aws-privateca-issuer:vv1.2.3": rpc error: code = Unknown desc = failed to pull and unpack image "public.ecr.aws/k1n1h4h4/cert-manager-aws-privateca-issuer:vv1.2.3": failed to resolve reference "public.ecr.aws/k1n1h4h4/cert-manager-aws-privateca-issuer:vv1.2.3": failed to do request: Head "https://public.ecr.aws/v2/k1n1h4h4/cert-manager-aws-privateca-issuer/manifests/vv1.2.3": dial tcp: lookup public.ecr.aws on 172.18.0.1:53: read udp 172.18.0.2:55707->172.18.0.1:53: i/o timeout
  Normal   BackOff    8s (x6 over 2m22s)   kubelet            Back-off pulling image "public.ecr.aws/k1n1h4h4/cert-manager-aws-privateca-issuer:vv1.2.3"
  Warning  Failed     8s (x6 over 2m22s)   kubelet            Error: ImagePullBackOff
```

**Comment by divyansh-gupta** [comment]
We released version v1.2.4 that solves this issue. Thank you.
